Update javascript-and-page-layout.md (#69609)
Azure/azure-content: articles/active-directory-b2c/javascript-and-page-layout.md

1. In your Azure AD B2C tenant, select **User flows**.
1. Select your policy (for example, "B2C_1_SignupSignin") to open it.

> 1. Select **Page layouts**. Choose one of the **Layout name**, then choose the **Page Layout Version (Preview)**.

Reviewer: @stevenxzhou a small suggestion, then this looks OK:
```suggestion
1. Select **Page layouts**. Choose a **Layout name**, and then choose the **Page Layout Version (Preview)**.
```
